tests/prettier/quotes.js

```diff
@@ -0,0 +1,46 @@
+// Every string will be changed to double quotes, unless we end up with fewer
```

Comment:
Could you put this one inside of tests/quotes/ this way it's going to test both --single-quote and not.